compiler/util/src/org/jetbrains/kotlin/config/LanguageVersionSettings.kt

```diff
@@ -282,6 +281,7 @@ enum class LanguageFeature(
     ForbidExtensionCallsOnInlineFunctionalParameters(KOTLIN_1_9, kind = BUG_FIX), // KT-52502
     ForbidInferringPostponedTypeVariableIntoDeclaredUpperBound(KOTLIN_1_9, kind = BUG_FIX), // KT-47986
     SkipStandaloneScriptsInSourceRoots(KOTLIN_1_9, kind = OTHER), // KT-52525
+    AssignOperatorOverloadForJvm(KOTLIN_1_9),
```

Comment:
Why the feature is bound to JVM? Probably it should work similarly for all backends. This also means that box tests should work for all backends.

Reply:
Yeah, I guess limiting the feature doesn't make sense. Removed the bound, renamed feature to `AssignOperatorOverload` and removed backend flags from tests.